src/OIGenerator.cpp

```diff
@@ -76,8 +76,20 @@ OIGenerator::findOilTypesAndNames(drgnplusplus::program& prog) {
       linkageName = linkageNameCstr;
     }
 
+    const std::string_view linkagePrefix =
+        "_ZN19ObjectIntrospection13getObjectSize";
+    if (!linkageName.starts_with(linkagePrefix)) {
+      LOG(INFO) << "expected symbol to match but missing linkagePrefix: `"
+                << linkageName << "`";
+      continue;
+    }
```

Comment:
Do we need to support both for backward compatibility? Should we have a TODO at some point to remove this?

Reply:
No, fortunately this code isn't widely used yet so no backwards compatibility is fine.